pkg/api/resource/typename_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"testing"
+)
+
+func TestStandardResourceTypes(t *testing.T) {
+	table := []struct {
+		input  string
+		result string
+	}{
+		{"memory", "kubernetes.io/resources/memory"},
```

Comment:
Sorry. Somewhere we discussed making this consistent with labels and annotations: `resources.kubernetes.io/memory`.